tensorboard/plugins/graph/tf_graph_common/op.ts

```diff
@@ -23,6 +23,9 @@ module tf.graph.op {
     'Acosh',
     'Add',
     'AddN',
+    'AdjustContrastv2',
```

Comment:
In most cases, `V2` is capitalized, but `AdjustContrastv2` looks right based on the name of the TensorFlow kernel. Just wanted to note that. FYI :)

Reply:
Acknowledged.